index.html

```diff
@@ -45,7 +45,7 @@ <h1 id="title">Survay Form</h1>
     Feel free to delete this message once you have read it.
     Good luck and Happy Coding!
 
-    - The freeCodeCamp Team
+    - The freeCodeCamp Team :)

```

Comment:
- [ ] I suggest you to add a profesional ReadMe file. You can use [this template.](https://github.com/microverseinc/readme-template) Remember that a ReadMe file should contain important information about your project like (description about it, how to set up the project, authors and aditional indications if needed).